includes/news/v2/mail/base.php

```diff
@@ -1,18 +1,110 @@
-	<?php echo get_template_part('includes/news/v2/mail/featured-stories'); ?>
-	<?php echo get_template_part('includes/news/v2/mail/events'); ?>
-	<?php echo get_template_part('includes/news/v2/mail/announcements'); ?>
-	<?php echo get_template_part('includes/news/v2/mail/footer'); ?>
 
+<body>
+	<table class="body">
+		<tr>
+			<td class="float-center" align="center" valign="top">
+				<center>
+					<table class="spacer float-center">
+						<tbody>
+							<tr>
+								<td height="16px" style="font-size:16px;line-height:16px;">&#xA0;</td>
+							</tr>
+						</tbody>
+					</table>
+					<table align="center" class="container float-center">
+						<tbody>
+							<tr>
+								<td>
+									<table class="spacer">
+										<tbody>
+											<tr>
+												<td height="16px" style="font-size:16px;line-height:16px;">&#xA0;</td>
+											</tr>
+										</tbody>
+									</table>
+									<table class="row">
+										<tbody>
+											<tr>
+												<th class="small-12 large-12 columns first last">
+													<table>
+														<?php echo get_template_part('includes/news/v2/mail/header'); ?>
+													</table>
+												</th>
+											</tr>
+										</tbody>
+									</table>
+									<table class="row">
+										<tbody>
+											<tr>
+												<th class="small-12 large-12 columns first last">
+													<table>
+														<?php echo get_template_part('includes/news/v2/mail/top-story'); ?>
+													</table>
+												</th>
+											</tr>
+										</tbody>
+									</table>
+									<table class="row">
+										<tbody>
+											<tr>
+												<th class="small-12 large-12 columns first last">
+													<table>
+														<?php echo get_template_part('includes/news/v2/mail/featured-stories'); ?>
+													</table>
+												</th>
+											</tr>
+										</tbody>
+									</table>
+									<table class="row">
+										<tbody>
+											<tr>
+												<th class="small-12 large-12 columns first last">
+													<table>
+														<?php echo get_template_part('includes/news/v2/mail/events'); ?>
+													</table>
+												</th>
+											</tr>
+										</tbody>
+									</table>
+									<table class="row">
+										<tbody>
+											<tr>
+												<th class="small-12 large-12 columns first last">
+													<table>
+														<?php echo get_template_part('includes/news/v2/mail/announcements'); ?>
+													</table>
+												</th>
+											</tr>
+										</tbody>
+									</table>
+									<table class="row">
+										<tbody>
+											<tr>
+												<th class="small-12 large-12 columns first last">
+													<table>
+														<?php echo get_template_part('includes/news/v2/mail/footer'); ?>
+													</table>
+												</th>
+											</tr>
+										</tbody>
+									</table>
+								</td>
+							</tr>
+						</tbody>
+					</table>
```

Comment:
I suggest we add the spacer at the bottom as well
```suggestion
					</table>
					<table class="spacer float-center">
						<tbody>
							<tr>
								<td height="16px" style="font-size:16px;line-height:16px;">&#xA0;</td>
							</tr>
						</tbody>
					</table>
```